generate-text-packs.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
I mostly kept them separate as to not edit the existing options for ref-packs generation. They are a bit more extensive (providing the option to specify TFM + to read execution options from `.csv`) which were not part of the original issue + I cannot assess their usefulness for text-only packages. On the other hand, there are only two scripts with minimal functionality which I don't think will see much change in terms of available options. With these two factors in mind, I kept the two scripts separate

If need be, I can unify their options and extend / move functionality as we see fit. Don't have a strong opinion on it

Reply:
I realize when we started down this journey it started as bringing in the existing installer logic but the more I look at this, it should be incorporated into the existing tooling rather than two distinct paths.  Beyond sharing the options the logic to restore the packages should be shared across both types as well.

Reply:
Sure, so to outline:

  - reference package and text-only package generation options should be unified (user can either manually specify package name and version together with optional inputs like TFM or feeds or they can provide a list of options to execute through a `.csv` or alike)
  - common code should be shared between the two projects (a good example is the restore procedure)
  - unlike existing reference package generation logic, the target one should behave the same way as the text-only one -> a thin wrapper delegating work to the project (point you made in your original comment)